requirements.txt

```diff
@@ -3,6 +3,7 @@ aiohappyeyeballs==2.4.0
 aiohttp==3.10.5
 aiosignal==1.3.1
 annotated-types==0.7.0
+anthropic==0.5.0  # Added Anthropic library
```

Comment:
I don't think we need to add anthropic to requirements.txt. This is managed by the aider-chat library.